src/solver/simulation/common-eco-adq.cpp

```diff
@@ -395,19 +383,17 @@ void BuildThermalPartOfWeeklyProblem(Data::Study& study,
         for (uint areaIdx = 0; areaIdx < nbPays; ++areaIdx)
         {
             auto& area = *study.areas.byIndex[areaIdx];
-            auto& tsIndex = NumeroChroniquesTireesParPays[numSpace][areaIdx];
             area.thermal.list.each([&](const Data::ThermalCluster& cluster)
             {
                     auto& Pt = problem.PaliersThermiquesDuPays[areaIdx]
                                .PuissanceDisponibleEtCout[cluster.index];
 
                     Pt.CoutHoraireDeProductionDuPalierThermique[hourInWeek] =
-                        cluster.getMarketBidCost(tsIndex.ThermiqueParPalier[cluster.areaWideIndex], hourInYear)
+                        cluster.getMarketBidCost(cluster.series->getSeriesIndex(hourInYear, year), hourInYear)
```

Comment:
```suggestion
                        cluster.getMarketBidCost(hourInYear, year);
```

Change the definition of `getMarketBidCost` to take (hour, year) instead of (hour, tsIndex).